### PR TITLE
Header levels note added

### LINKED ISSUE #941: Add “creating new section” steps to readme
Describe steps for creating both L1-3 and L4 sections to README.

### PR DESCRIPTION
**Pull request by har00ga**
Fixes #941 

This adds a note explaining how subsection numbering is handled within the spec build. I'm not sure if this extends to a L5 as well, confirmation on that would be great. The location may be a little awkward as well, thoughts? @mishmosh

### CODE REVIEW

**Comment by mishmosh** [comment]
Can you round this out with a bit more detail? (I'm thinking of details like to add L1-L3 content you have to create a new `.md` file _and_ a new listing in the parent index file; whereas to add L4 content you only need to add a header.)

**Comment by mishmosh** [comment]
As for location, you're right, there's not an ideal place. Maybe https://filecoin-project.github.io/specs/#intro__process__contributing ?

**Comment by har00ga** [comment]
Let me know what you think of the new changes @mishmosh , and good call on the location -- happy to put a copy there once we're happy with the content!

File: README.md [hunk 1]
```diff
@@ -96,6 +96,14 @@ For every subsystem in the Filecoin spec, it must always have a markdown file th
 <img src="src/diagrams/buildsys/buildsys.dot.svg" width="50%">
 -->
 
+#### Adding L1 - L4 sections
+
+The specification is broken down into 4 levels (`#.#.#.#`). The L1 and L2 numbers in this sequence are determined by the first two directories extending from `/src/`; for example, `/src/systems/filecoin_mining/` resolves to `2.6.`. 
+
+The L3 number is generated by creating an additional directory within a L2 directory, containing it's own appropriately formatted `index.md`. This new directory name must then be added to the `entries` field of the L2 `index.md` file, sequentially ordered as they are to be within the specification.
```
Comment (mishmosh) [inline README.md:103]:
```suggestion
The L3 number is generated by creating an additional directory within a L2 directory, containing its own appropriately formatted `index.md`. This new directory name must then be added to the `entries` field of the L2 `index.md` file, sequentially ordered as they are to be within the specification.
```

File: README.md [hunk 2]
```diff
@@ -96,6 +96,14 @@ For every subsystem in the Filecoin spec, it must always have a markdown file th
 <img src="src/diagrams/buildsys/buildsys.dot.svg" width="50%">
 -->
 
+#### Adding L1 - L4 sections
```
Comment (mishmosh) [inline README.md:99]:
```suggestion
#### Adding new sections
```

**Pull request merged**